endpoints.json

```diff
@@ -52,6 +52,7 @@
   "PATCH /api/reviews/:review_id": {
     "description": "Patches a single review vote by review_id.",
     "queries": [],
+    "exampleRequest": {"key": "inc_votes: 1" },
```

Comment:
You need it without the key - literally "exampleRequest": {"inc_votes": 1 }

Same for the other one